docs/user/learner.rst

```diff
@@ -86,15 +125,14 @@ To play videos in **Kolibri** you have several available control buttons at the
 
 * Play/Pause buttons
 * Rewind/Fast forward buttons by +/- 10 seconds
-* Time tracker indicator with scrollbar
+* Time tracker indicator with progress bar
 * Video duration indicator
-* Mute sound button
 * Volume scrollbar
 * Playback speed selector
 * Fullscreen button
 
 
-Use the **Download Media** button below the video player to download the video file to your computer.
+Use the **Download content** button below the video player to download the video and thumbnail files to your computer. On some videos you will have an option to download videos in high and low resolution.
```

Comment:
> On some videos you will have an option to download videos in high and low resolution

would require a comma:
"On some videos, you will have an option to download videos in high and low resolutions"

Personally, I'd rather do:
"Some videos will even provide multiple resolution options".